### PR TITLE
Add Spanish translations

### PR DESCRIPTION
**Pull request by inkel**
I've tried to make the translations as generic-Spanish as possible. I've found a duplicate (`sponsors`) so I've removed.

I haven't translated yet `venue` as I'm looking for the proper word in Spanish for it. In Argentina we would very likely use venue, but that's definitely not the right word.

Last but not least, I haven't translated yet `under license` as I'd like to know more about the context of where it is used, as it could mean _without license_ or _using the following license_.

### CODE REVIEW

File: data/translations/es.yaml [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
-where_say: Where to stay
-desc_hotels: We selected some hotels next to the venue.
-value_hotel_canasvieiras: From U$ 50 per night
-soon_hotels: Soon more hotels...
-sponsors: Sponsors
-companies_sponsors: Companies that are helping make the GopherCon Brasil a reality.
-registration: REGISTER HERE
-character_copyright: Gopher character is based on a sketch by
+date: "27, 28 y 29 de Septiembre de 2018"
+text_desc: "El mayor evento dedicado a Go de Brasil."
```
Comment (jeffprestes) [inline data/translations/es.yaml]: Aquí @inkel puedes escribir "El mayor evento dedicado a Go en America Latina" ;)

File: data/translations/es.yaml [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
+text_center: "GopherCon Brasil 2018 tendrá lugar del 27 al 29 de Septiembre (en el International Hotel Canasvieiras),
```
Comment (jeffprestes) [inline data/translations/es.yaml]: Aquí un error en original. El texto correcto és "tendrá lugar del 28 al 29 de Septiembre"

File: data/translations/es.yaml (hunk 1 above)
Comment (inkel) [reply data/translations/es.yaml]: 👍 me gusta más 😸

File: data/translations/es.yaml (hunk 2 above)
Comment (inkel) [reply data/translations/es.yaml]: Ya me parecía. Excelente, corrigiendo.

**Pull request merged**